services/blockchain-coordinator/app.js

```diff
@@ -22,13 +22,13 @@ const {
 } = require('lisk-service-framework');
 
 const config = require('./config');
-
-LoggerConfig(config.log);
-
 const packageJson = require('./package.json');
-const { setAppContext } = require('./shared/utils/request');
+
 const { init } = require('./shared/scheduler');
+const { setAppContext } = require('./shared/utils/request');
+const { waitForNodeToFinishSync } = require('./shared/init');
 
+LoggerConfig(config.log);
```

Comment:
Do we still need this?

Reply:
Yes, to set the application name and other necessary meta information for the logger.